mysql-test/mysql-test-run.pl

```diff
@@ -6649,7 +6680,7 @@ sub valgrind_arguments {
 #
 
 sub valgrind_exit_reports() {
```

Comment:
After the change in this function MTR prints a a big number of
```
Argument "" isn't numeric in exit at /home/yura/ws/percona-server/mysql-test/mysql-test-run.pl line 1055.
```
errors when run with the `--valgrind` option